src/backend/metal/src/command.rs

```diff
@@ -1756,6 +1775,17 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
         if let Some(ref color) = pipeline.baked_states.blend_color {
             commands.push(self.set_blend_color(color));
         }
+
+        let depth_bias = pipeline.rasterizer.depth_bias.unwrap_or_else(|| {
+            pso::DepthBias {
+                const_factor: 0.0,
```

Comment:
Not sure if I should be doing this here or somewhere else

Reply:
seems fine, although could be simpler by doing `unwrap_or_default`